frontend/src/scenes/persons/MergePerson.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from 'react'
+import api from 'lib/api'
+import { Button, Select } from 'antd'
+import Modal from 'antd/lib/modal/Modal'
+import { PersonType } from '~/types'
+import { useActions, useValues } from 'kea'
+import { personsLogic } from './personsLogic'
+import { toast } from 'react-toastify'
+import posthog from 'posthog-js'
+
+export function MergePersonButton({
+    person,
+    onPersonChange,
+}: {
+    person: PersonType
+    onPersonChange: CallableFunction
+}): JSX.Element {
+    const [isModalOpen, setIsModalOpen] = useState(false)
+    const [selectedPeople, setSelectedPeople] = useState([])
+
+    const { loadPersons, setListFilters } = useActions(personsLogic)
+    const { persons } = useValues(personsLogic)
+    useEffect(() => {
+        loadPersons()
+    }, [])
+
+    useEffect(() => {
+        loadPersons()
+    }, [person.distinct_ids, person.id])
+    return (
+        <>
+            {isModalOpen && (
+                <Modal
+                    visible
+                    title={'Merge ' + person.name}
+                    onCancel={() => setIsModalOpen(false)}
+                    onOk={async () => {
+                        const newPerson = await api.create('api/person/' + person.id + '/merge/', {
+                            ids: selectedPeople,
+                        })
+                        if (newPerson.id) {
+                            toast('People succesfully merged into one.')
+                            posthog.capture('merge person completed')
+                            setIsModalOpen(false)
+                            onPersonChange(newPerson)
+                        }
+                    }}
+                >
+                    Merge all properties and events of the selected users into <strong>{person.name}.</strong>
+                    <br />
+                    <br />
+                    Note: If there is a conflict, the properties of <strong>{person.name}</strong> will take precedence.
+                    <Select
+                        mode="multiple"
+                        allowClear
+                        showSearch
+                        style={{ width: '100%' }}
+                        placeholder="Please select people to merge"
+                        onChange={(value) => setSelectedPeople(value)}
```

Comment:
Nit: My TS linter complains about `value` being the wrong type

It works, so it could be passed as any just to silence it

Reply:
I think this is an issue with ant.d. Normally it would be a string but in this case it's an array because `multiple`

Reply:
yeah I mean not much to do unless you want to pass it as `any`